### DISCUSSION TITLE
docs/ninjutsu/low-ranking-ninja/think-of-wonderful-things

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#23] [General]
# docs/ninjutsu/low-ranking-ninja/think-of-wonderful-things

就算头顶有个摄像头，也能摸鱼！没有人知道你坐在工位的时候在想什么，何不直接开始发呆？

https://996.ninja/docs/ninjutsu/low-ranking-ninja/think-of-wonderful-things

**Comment by wududu0920** [comment]
这就是我QAQ

**Reply by MingGH** [reply]
说说你的发呆幻想

**Reply by wududu0920** [reply]
针对老板的刺杀计划，重生之我是火星人间谍，丧尸求生，要不多抽象有多抽象，及其考验想象力和中二能力（）
所以说你这么高强度冲浪是在摸鱼吗（）

**Reply by MingGH** [reply]
这么炫酷！放假呢，不摸鱼，直接骑鲸鱼头上冲浪

**Reply by yuanaei** [reply]
真实……

**Comment by lg961126** [comment]
我是谁，我是否存在，我是以什么形态存在，我如何被观测到，如何证明我的存在。

**Reply by MingGH** [reply]
当你思考的时候，你就存在

**Reply by lg961126** [reply]
太主观了（）

**Reply by butaiguaizzz** [reply]
我思故我在是吧

**Comment by liuhenlab** [comment]
还有一些其他的，比如思考减肥方案，健身方案，菜谱，这周食材的搭配，周末去哪玩，硬生生给我整成j人了

**Reply by MingGH** [reply]
好好好
